css/input.css

```diff
@@ -14,15 +14,15 @@
 
 .input-div {
     width: 80%;
-    margin: var(--size-0-75) var(--size-0-75);    
+    margin: var(--size-0-75) var(--size-0-75);
 }
 
 .input-label {
     padding: var(--size-0-5) 0;
     display: block;
 }
 
-.input-field:disabled{
+.input-field:disabled {
     background-color: var(--grey-bg);
```

Comment:
the same suggestion here for color variable naming

Reply:
Thanks a lot !